### PR TITLE
[DFFA-156] Build a To-Do App with REACT

### PR DESCRIPTION
**Pull request by DevOge**

### CODE REVIEW

**Comment by Ireti203** [comment]
Attach screenshots of your work

**Comment by DevOge** [comment]
reactlook

File: submissions/OgeEjim/React/todo-list/src/component/overall.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                        onChange={this.handleChange}
+                        placeholder="create a new todo..."
+                    />
+
+                    <button className="btn btn-primary" onClick={(e) => this.handleCreate(e)} disabled={this.state.inactive} >+</button>
+                </div>
+                <div className="mt-4 listContainer">
+                    <div className="mt-4">
+                        {
+                            this.state?.list?.map((todo, index) => <Card onButtonClick={this.handleDone} index={index} todo={todo} key={index} />)
```
Comment (ToluAkin) [inline submissions/OgeEjim/React/todo-list/src/component/overall.js:76]: What are you trying to do here `this.state?.list?` ? @DevOge

**Review by ToluAkin — CHANGES_REQUESTED** [review]
Well done @DevOge

**Comment by DevOge** [comment]
Okay, good evening
.. I haven't quite wrapped my head around react or js yet tho.. I did this
by following tutorials and meeting people, the ? is supposed to help with
errors. I can take it out ... just found out it's not necessary.

**Pull request closed without merging**